### DISCUSSION TITLE
Root Toggle items not having "items-center" class

### DISCUSSION THREAD

**Discussion by Skreeauk** [fuma-nama/fumadocs#758] [Q&A]
For Root Toggle main button, theres a "items-center" so if the image/logo is big, the text is still centered. While the items inside does not have "items-center".

Would like to know if theres any solution beside making the image/logo smaller?

**Comment by fuma-nama** [comment]
I think the current style looks good:
![image](https://github.com/user-attachments/assets/4c83c06d-0ee7-406a-8f60-dea60bb1d21b)

Maybe you can add some margins to the title node (using `span`).

**Reply by Skreeauk** [reply]
Yeah thats fine if the image/logo is smaller but what I wanted to use has really bad quality when its small. Guess I'll just use another logo. Thanks for responding

**Reply by nktnet1** [reply]
coming from #823 - would be nice if this was configurable, or if there's a way to make the selected display and dropdown looks the same (i.e. either both top-aligned or both middle-aligned) :smile:.

**Reply by nktnet1** [reply]
Yay, looks like it was fixed in [fumadocs-ui@13.4.8](https://github.com/fuma-nama/fumadocs/releases/tag/fumadocs-ui%4013.4.8). Thanks @fuma-nama <3!
